blackbox/docs/enterprise/index.rst

```diff
@@ -4,9 +4,28 @@
 Enterprise Features
 ===================
 
-CrateDB provides an `Enterprise Edition`_, which is activated by default. The
-source code remains open, but if you want to use one of the enterprise features
-in production you need to purchase a `enterprise license`_.
+CrateDB provides an `Enterprise Edition`_. The source code remains open, but if
+you want to use one of the enterprise features in production you need to
+purchase a `enterprise license`_.
+
+The `Enterprise Edition`_ is enabled by default, the
+:ref:`license.enterprise <conf-node-enterprise-license>` config being true, for
+a period of 30 days that will allow users to evaluate the enterprise features.
+
+Upon the expiration of the 30 days trial period you will have to request an
+enterprise license by `contacting our sales department`_.
+Setting a new license is achieved using the :ref:`SET LICENSE <ref-set>`
+statement.
+
+Alternatively, users seeking to use the community edition can do so at any
+point by disabling the enterprise license using the
+:ref:`license.enterprise <conf-node-enterprise-license>` config.
+
+.. WARNING::
+
+  CrateDB will continue to run after the license expires but only the ``SET``
```

Comment:
```suggestion
  CrateDB will continue to run after the license expires but only the ``SET LICENSE``
```

Or do we allow all SET statements?

Reply:
👍 made the change (one `SET LICENSE` is allowed)